experimental/traffic-portal/src/app/core/core.module.ts

```diff
@@ -92,10 +94,13 @@ export const ROUTES: Routes = [
 	{ component: CoordinatesTableComponent, path: "coordinates" },
 	{ component: TypesTableComponent, path: "types" },
 	{ component: TypeDetailComponent, path: "types/:id"},
+	{ component: ParametersTableComponent, path: "parameters" },
+	{ component: ParameterDetailComponent, path: "parameters/:id" },
 	{ component: StatusesTableComponent, path: "statuses" },
 	{ component: StatusDetailsComponent, path: "statuses/:id" },
 	{ component: ISOGenerationFormComponent, path: "iso-gen"},
 	{ component: ProfileTableComponent, path: "profiles"},
+	{ component: ProfileTableComponent, path: "profiles/param/:id"},
```

Comment:
I don't believe this route is actually necessary. You can just implement a custom query string parameter on the profiles table at the normal `profiles` route in pretty much exactly the same way.